lib/galaxy_test/api/test_workflows.py

```diff
@@ -1347,9 +1347,8 @@ def test_run_subworkflow_simple(self) -> None:
             assert sum(1 for step in steps if step['subworkflow_invocation_id'] is None) == 3
             subworkflow_invocation_id = [step['subworkflow_invocation_id'] for step in steps if step['subworkflow_invocation_id']][0]
             subworkflow_invocation = self.workflow_populator.get_invocation(subworkflow_invocation_id)
-            # inner_input should be step 0, random_lines should be step 1, but step order gets lost on python < 3.6
```

Comment:
> but step order gets lost on python < 3.6

I think that's not true, invocation steps are not sorted currently, which means API tests with this change are failing! I'll see if it's to fix or I'll revert.